mqttwarn/core.py

```diff
@@ -627,6 +628,10 @@ def connect():
     # Update our runtime context (used by functions etc) now we have a connected MQTT client
     context.invoker.srv.mqttc = mqttc
 
+    # if publishversion configuration entry exists, use it to publish the current mqttwarn version
+    if cf.publishversion:
```

Comment:
Did you make sure mqttwarn does not croak when accessing `cf.publishversion` without that configuration item being defined in `mqttwarn.ini`?

Reply:
actually no, now that I think about it, it will cause an exception.  As publishversion will not exist and not return a none just because it's not there.